rest-api/pom.xml

```diff
@@ -47,7 +47,8 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.4</version> <!-- allows consuming the new APIs -->
+            <!-- Temp version until final is released -->
+            <version>2.12-alpha</version>
```

Comment:
Be super careful with `alpha`/`beta` dependencies. `maven-release-plugin` will let you go ahead and release this plugin as is—but if anyone actually tries to install it from the non-experimental update center, Jenkins will happily let you go ahead, and then issue an error when you restart. Oops.

So I tend to say that you should only use (timestamped) `SNAPSHOT` dependencies until there is a full upstream release.